html/index.html

```diff
@@ -48,6 +48,7 @@
                     <div id="phone-icons">
                         <i class="fas fa-signal" id="phone-signal" data-toggle="tooltip" data-placement="left" title="QBCore 5G"></i>
                         <i class="fas fa-battery-three-quarters" id="phone-battery"></i>
+                        <i class="fas fa-hashtag" style="padding-right: 10px;" id="phone-serverid" data-toggle="tooltip" data-placement="left" title="Server ID"></i>
```

Comment:
Should definitely look at a better way to apply styles here, look at the clock code inside the phone for more details.